packages/auth/src/accessToken/routes.test.ts

```diff
@@ -202,9 +202,8 @@ describe('Access Token Routes', (): void => {
           client: TEST_CLIENT,
           jwk: testJwk
         } as ClientKey)
-      const now = new Date(new Date().getTime() + 4000)
-      jest.useFakeTimers()
-      jest.setSystemTime(now)
+      const now = new Date(new Date().getTime() + 4000 * 1000)
```

Comment:
It might be clearer to make this relative to `token`'s `createdAt` + `expiresIn`